roles/bindle-profiles/tasks/main.yml

```diff
@@ -16,26 +16,31 @@
   git: repo=https://github.com/CloudBindle/Bindle
        dest=/home/{{ user_name }}/architecture2/Bindle
        version={{ bindle_git_branch }}
+       force=no
```

Comment:
Will this fail silently if there are changes in their repo? Make sure that the script fails loudly if thy have changes